src/pages/Home/index.jsx

```diff
@@ -1,5 +1,11 @@
+import WhyChooseUs from "../../components/whyChooseUs";
```

Comment:
Só remove essa importação, deixa ela só quando vc estiver codando e testando, ai quando for fazer o push sempre remova as importações, pra deixar as páginas, por enquanto, somente com o return padrão.